pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -36,6 +37,10 @@ var (
 	MetricResType = common.MetricResTypeNetworkInfo
 )
 
+const (
+	VpcPathMinSegments = 8
```

Comment:
I might miss something, does this path `/orgs/default/projects/project-quality/vpcs/xxx` only have 7 segments?

Reply:
you're right, I made a mistake.